bot/exts/info/help.py

```diff
@@ -173,12 +173,16 @@ async def command_formatting(self, command: Command) -> Embed:
         if aliases:
             command_details += f"**Can also use:** {aliases}\n\n"
 
-        # when command is disabled, show message about it
+        # when command is disabled, show message about it,
+        # when other CommandError instance is raised, log warning about it
         # otherwise check if the user is allowed to run this command
-        if not command.enabled:
+        try:
+            if not await command.can_run(self.context):
+                command_details += "***You cannot run this command.***\n\n"
+        except DisabledCommand:
             command_details += "***This command is disabled.***\n\n"
-        elif not await command.can_run(self.context):
-            command_details += "***You cannot run this command.***\n\n"
+        except CommandError as e:
+            log.warning(f"An exception raised when trying to check {command.name} command running permission: {e}")
```

Comment:
Some checks legitimately raise `CommandError` to customise the way in which the check fails. Therefore, if a `CommandError` is raised, it should be treated like a check failure (i.e. this user cannot run the command), rather than some internal error that should be logged.